import_export/management/commands/import_file.py

```diff
@@ -0,0 +1,137 @@
+from __future__ import unicode_literals
+
+import mimetypes
+import argparse
+
+try:
+    FileNotFoundError
+except NameError:
+    FileNotFoundError = IOError
+
+try:
+    from django.apps import apps as django_apps
+except ImportError:
+    from django.db import models as django_apps
+
+try:
+    from django.utils.module_loading import import_string
+except ImportError:
+    from django.utils.module_loading import import_by_path as import_string
+
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+from django.utils.encoding import force_text
+from django.utils.translation import ugettext as _
+
+from import_export.formats import base_formats
+from import_export.resources import modelresource_factory
+
+
+FORMATS = {
+    None: base_formats.CSV,
```

Comment:
Does `None` here means that CSV is *default* format?

Reply:
Yes, I have time to try making it a configurable variable, should I?